lib/web/command_test.go

```diff
@@ -123,13 +139,87 @@ func TestExecuteCommandHistory(t *testing.T) {
 	require.Equal(t, "node", result.NodeID)
 }
 
+func TestExecuteCommandSummary(t *testing.T) {
+	t.Parallel()
+
+	openAIMock := mockOpenAISummary(t)
+	openAIConfig := openai.DefaultConfig("test-token")
+	openAIConfig.BaseURL = openAIMock.URL + "/v1"
+	s := newWebSuiteWithConfig(t, webSuiteConfig{
+		disableDiskBasedRecording: true,
+		OpenAIConfig:              &openAIConfig,
+	})
+	authPack := s.authPack(t, "foo")
+
+	ctx := context.Background()
+	clt, err := s.server.NewClient(auth.TestUser("foo"))
+	require.NoError(t, err)
+
+	// Create conversation, otherwise the command execution will not be saved
+	conversation, err := clt.CreateAssistantConversation(context.Background(), &assist.CreateAssistantConversationRequest{
+		Username:    "foo",
```

Comment:
nit: I'd extract the "foo" user to a constant to make it clear that this is the same user and not a random unrelated string.